.github/workflows/release.yml

```diff
@@ -1,62 +1,20 @@
-name: Deployment
+name: Publish Release
 
 on:
-  release:
-    types: [ published ]
+  workflow_dispatch:
+  repository_dispatch:
+    types: [ stdlib-release-pipeline ]
```

Comment:
is it correct to have `stdlib-release-pipeline` repository dispatch as a release trigger for the connector modules? @MohamedSabthar @ThisaruGuruge any thoughts?

Reply:
We can remove this.

Reply:
removed [70dfeb8](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/322/commits/70dfeb8885ce88fef103f10126fbef96e80e9ff2)